go/cmd/dolt/commands/schcmds/import.go

```diff
@@ -58,9 +58,13 @@ const (
 
 var MappingFileHelp = "A mapping file is json in the format:" + `
 
-	{
-		"source_field_name":"dest_field_name"
-		...
+	{ "rows": 
+		[
+			{
+				"source_field_name":"dest_field_name"
+				...
+			}, ...
+		]
```

Comment:
This is good content, but I think it's in the wrong section... the mapping file maps names between the input file and the destination schema to rename items, but we want to add some description of the input file.